src/strings/en-us.json

```diff
@@ -1607,5 +1607,14 @@
     "UnknownVideoStreamInfo": "The video stream info is unknown",
     "UnknownAudioStreamInfo": "The audio stream info is unknown",
     "DirectPlayError": "There was an error starting direct playback",
-    "SelectAll": "Select All"
+    "SelectAll": "Select All",
+    "Clip": "Featurette",
+    "Trailer": "Trailer",
+    "BehindTheScenes": "Behind the scenes",
```

Comment:
I think title case might work better for these.

Reply:
Pushed.